### PR TITLE
Change terminology `Sleeping` to `Archived`

### PR DESCRIPTION
**Pull request by avinassh**
Some users are getting confused with the terminology `Sleeping` as we also have cold starts. This patch changes the word to `Archived`. This also changes the word `Wake up` to `Unarchive`

CLI PR: https://github.com/tursodatabase/turso-cli/pull/902

API: https://github.com/tursodatabase/turso/pull/1998

### CODE REVIEW

File: api-reference/openapi.json [hunk 1]
```diff
@@ -2232,7 +2232,7 @@
           },
           "sleeping": {
```
Comment (notrab) [inline api-reference/openapi.json:2233]: @athoscouto @avinassh do we plan to update the API too? Otherwise it’s just as confusing if we don’t.

Comment (avinassh) [reply api-reference/openapi.json:2233]: I have not made changes to the API to keep backward compatibility. 

I will send a follow up patch and update the API spec and replace `sleeping` with `archived`. On the API server, I will send both the fields. WDYT?

Comment (notrab) [reply api-reference/openapi.json:2233]: Sending both fields sounds good to me. We can then update the docs to use archived and never speak of sleeping again 😂

Comment (avinassh) [reply api-reference/openapi.json:2233]: what is `sleeping`

**Comment by avinassh** [comment]
The server now sends `archived` field (https://github.com/tursodatabase/turso/pull/1999). So I have updated the Open API spec

File: cli/db/wakeup.mdx
Comment (notrab) [inline cli/db/wakeup.mdx:1]: Do we need to rename this path? Also, we should add a redirect for the old pages to the new ones. I think mint.json has an example of this, on my phone though so unable to test locally.

Comment (avinassh) [reply cli/db/wakeup.mdx:1]: I just renamed to keep it consistent. However, you are right about redirects and I missed that. I have added redirects now. However, if you want to keep the old path names, then I can revert those changes. 

I am having some issue with node locally and not able to run it. I will report back if I get it to working.

Comment (notrab) [reply cli/db/wakeup.mdx:1]: The changes look good!

**Review by notrab — APPROVED** [review]

**Pull request merged**